src/utils/deepEqual.ts

```diff
@@ -0,0 +1,25 @@
+// https://dmitripavlutin.com/how-to-compare-objects-in-javascript/
+export function deepEqual(object1: any, object2: typeof object1): boolean {
```

Comment:
Should this be part of this PR?

Reply:
Damn, it's an artifact after merging. Thanks